pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -144,16 +145,21 @@ func (p *pvcEvaluator) GroupKind() schema.GroupKind {
 // Handles returns true if the evaluator should handle the specified operation.
 func (p *pvcEvaluator) Handles(a admission.Attributes) bool {
 	op := a.GetOperation()
-	if op == admission.Create {
+	initialized, err := initialization.IsObjectInitialized(a.GetObject())
+	if err != nil {
+		// fail closed, will try to give an evaluation.
+		return true
+	}
+	if op == admission.Create && initialized {
```

Comment:
unconditionally handle create

Reply:
also pvc uses object count, so we'd want to handle an uninitialized create

Reply:
I updated it to be just like the pod evaluator.